core/src/main/resources/filodb-defaults.conf

```diff
@@ -338,3 +338,53 @@ kamon.zipkin {
   max.requests = 128
   message.max.bytes = 262144
 }
+
+kamon {
+  prometheus.buckets {
+    # Have more buckets, better resolution really helps esp with heatmaps
+    default-buckets = [
+      4,
+      8,
+      16,
+      32,
+      64,
+      128,
+      256,
+      512,
+      1024,
+      2048,
+      4096,
+      8192,
+      16384,
+      32768,
+      65536,
+      128000,
+      256000,
+      512000
+    ]
+
+    # Start at 0.1ms so we can measure smaller timings
+    time-buckets = [
+      0.0001,
```

Comment:
Shouldnt we go lower since we measure sub-ms latencies like actor mailbox time, etc. 
Our overall query latency is a few ms, and we have several spans within. So we should start lower.

Reply:
Yes this starts at 0.1ms, but I can make it go even lower.

@vishramachandran BTW the overall lesson is that useful buckets often need more than 20 buckets.  We should expect 30-40 buckets for histograms.